instructor-notes/solutions/fileexists_download-fastq.sh

```diff
@@ -55,9 +58,18 @@ if [ ! -e "$FASTQ_DEST/$FASTQ_R2" ]; then
 fi
 
 # Explore: how many lines are in the file?
+echo "The number of lines in $FASTQ_R2 is:"
```

Comment:
Can you also add this to `variables_download-fastq.sh` while you're at it?

Reply:
done